app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
I'll leave it up to you.  I feel the same about details or attributes. :)

Generally, when I see an object (e.g. subscription) that is a hash, I assume that the contents of the hash are attributes/details/characteristics of the object; therefore, for me the additional clarification in the name is not needed. But, that is only one devs opinion.  :)